people/views.py

```diff
@@ -70,6 +70,28 @@ def profile(request, year, pk):
 		'roles': RecruitmentApplication.objects.filter(user = user, status = 'accepted').order_by('recruitment_period__fair')
 	})
 
+def profile_delete(request, year, pk):
+	fair = get_object_or_404(Fair, year = year)
+	user = get_object_or_404(User, pk = pk)
```

Comment:
`pk`-variable can be retrieve from request. So remove pk
Here is an example.
https://github.com/armada-ths/ais/blob/6a0856c5e3d1bf575a1e6091c5077e75327c5637/kth_login/views.py#L35

so for you maybe it could be:
```python
user = request.session['user']
```